pulp_rpm/app/models/repository.py

```diff
@@ -440,8 +441,27 @@ class RpmDistribution(Distribution):
     def content_handler(self, path):
         """Serve config.repo and public.key."""
         if path == self.repository_config_file_name:
+            publication = None
```

Comment:
@gerrod3 Could you take this logic and put it in a method named `get_publication()`?